### ISSUE TITLE
Is it necessary to upload the clones to git?

### ISSUE THREAD

**Issue by guplem** [314pies/ParrelSync#4]
I do not know if it is necessary to keep the clones updated, or if they just reference the same assets as the main project and nothing will happen if receive an update of the project that doesn't include the (updated) clones.

I added this to my .gitignore: `*_clone_[0-9]*/`

So now, no cloned projects are being synchronized (tracked by git) for me but I'm not sure about if what I'm doing is properly done.

**Comment by 314pies** [comment]
Aha, here come the first issue(not even finish the readme yet!)

Cloned projects don't need to be tracked by git, and it's safe(and probably better) to exclude it by adding it to .gitignore.
Let me know if you have any more question.

And just being curious, how did you find this project?

**Comment by guplem** [comment]
Nice! Thanks!

I will use this project in a small video game that a group of university mates are developing. Due to Covid-19, we had to transform it from a local multiplayer to an online multiplayer. So, actually, ParrelSync is helping a lot!

Maybe this project can be used to test a little bit this tool.

I found out about ParrelSync while I was investigating about uEcho, a project that at first sight seems very similar to this one.
So [in a forum post](https://forum.unity.com/threads/editor-extension-multiple-unity-editors-same-project.417077/#post-5815249) I found a link to this GitHub repo.

Thanks for your work!
